### ISSUE TITLE
Can we export to docnow catalog

### ISSUE THREAD

**Issue by kayiwa** [DocNow/docnow#67]
Is it possible to export directly (?) from your own docnow instance to docnow catalog.

**Comment by edsu** [comment]
Yes, this is definitely something we've talked about doing in Phase 2. I think it will involve making the catalog into more of a webapp, and less of a static site deployed w/ GitHub pages. But it would be cool if there ways of getting around that.
